### ISSUE TITLE
ModuleNotFoundError: No module named 'pangocffi._generated'

### ISSUE THREAD

**Issue by arjun-kava** [leifgehrmann/pangocffi#22]
OS: macOS Catalina
OS version: 10.15.7

While installing using python@3.9, It is throwing the error: 

```py
ERROR: Command errored out with exit status 1:
     command: /usr/local/opt/python@3.9/bin/python3.9 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/tx/3w2tbvmn64n0xd6ktnxf1yqc0000gn/T/pip-install-p5relybu/pangocairocffi/setup.py'"'"'; __file__='"'"'/private/var/folders/tx/3w2tbvmn64n0xd6ktnxf1yqc0000gn/T/pip-install-p5relybu/pangocairocffi/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/tx/3w2tbvmn64n0xd6ktnxf1yqc0000gn/T/pip-pip-egg-info-ce9ywn4e
         cwd: /private/var/folders/tx/3w2tbvmn64n0xd6ktnxf1yqc0000gn/T/pip-install-p5relybu/pangocairocffi/
    Complete output (27 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/tx/3w2tbvmn64n0xd6ktnxf1yqc0000gn/T/pip-install-p5relybu/pangocairocffi/setup.py", line 9, in <module>
        setup(
      File "/usr/local/lib/python3.9/site-packages/setuptools/__init__.py", line 153, in setup
        return distutils.core.setup(**attrs)
      File "/usr/local/Cellar/python@3.9/3.9.0_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/core.py", line 108, in setup
        _setup_distribution = dist = klass(attrs)
      File "/usr/local/lib/python3.9/site-packages/setuptools/dist.py", line 423, in __init__
        _Distribution.__init__(self, {
      File "/usr/local/Cellar/python@3.9/3.9.0_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/dist.py", line 292, in __init__
        self.finalize_options()
      File "/usr/local/lib/python3.9/site-packages/setuptools/dist.py", line 695, in finalize_options
        ep(self)
      File "/usr/local/lib/python3.9/site-packages/setuptools/dist.py", line 702, in _finalize_setup_keywords
        ep.load()(self, ep.name, value)
      File "/usr/local/lib/python3.9/site-packages/cffi/setuptools_ext.py", line 219, in cffi_modules
        add_cffi_module(dist, cffi_module)
      File "/usr/local/lib/python3.9/site-packages/cffi/setuptools_ext.py", line 49, in add_cffi_module
        execfile(build_file_name, mod_vars)
      File "/usr/local/lib/python3.9/site-packages/cffi/setuptools_ext.py", line 25, in execfile
        exec(code, glob, glob)
      File "pangocairocffi/ffi_build.py", line 11, in <module>
        from pangocffi.ffi_instance_builder import \
      File "/usr/local/lib/python3.9/site-packages/pangocffi/__init__.py", line 2, in <module>
        from ._generated.ffi import ffi
    ModuleNotFoundError: No module named 'pangocffi._generated'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by leifgehrmann** [comment]
This will be related to https://github.com/leifgehrmann/pangocairocffi/issues/8

Hopefully https://github.com/leifgehrmann/pangocffi/pull/21 will fix this issue, which I aim to finish testing tomorrow.

**Comment by leifgehrmann** [comment]
Hi @arjun-kava, I've published a new release for pangocffi (v0.8.0) and pangocairocffi (v0.4.0). Can you try installing again?

**Comment by arjun-kava** [comment]
Hey, @leifgehrmann, Thank you for your fast response. It is working fine now.
